src/components/ColumnHeaderToolTip/ColumnHeaderToolTip.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import theme from '../../theme'
+
+export const TooltipPopup = styled('span')`
+  display: block;
+  max-width: 25rem;
+  width: max-content;
+  background: ${theme.color.primaryColor};
+  color: ${theme.color.white};
+  position: absolute;
+  font-size: ${theme.typography.smallFontSize};
+  clip-path: polygon(0% 0%, 100% 0%, 100% 83%, 53% 83%, 47% 100%, 40% 83%, 0 83%);
```

Comment:
I don't really like using percentages for the triangle piece, but I kind of suck at polygon clip paths and was struggling to make it precise. Maybe Al is better at this!

Reply:
<img width="290" alt="Screen Shot 2023-07-21 at 12 48 51 PM" src="https://github.com/data-mermaid/mermaid-webapp/assets/26089140/63ffa8c4-77af-4cd7-b519-19c40dc0f174">